dali/python/nvidia/dali/fn.py

```diff
@@ -44,13 +46,13 @@ def _to_snake_case(pascal):
                 if len(out) > 0:
                     out += '_'
                 if nupper > 1:
-                    out += pascal[start:i-1].lower() + '_'
-                out += pascal[i-1].lower()
+                    out += pascal[start:i - 1].lower() + '_'
```

Comment:
That's just terrible. Now the `start:i` and `1` look like the terms of the subtraction :(